agent/statemanager/state_manager.go

```diff
@@ -48,6 +48,7 @@ const (
 	//   b) Add 'ContainerResourcesProvisioned' as a new 'ContainerStatus' enum
 	//   c) Add 'SteadyStateStatus' field to 'Container' struct
 	//   d) Add 'ENIAttachments' struct
+	//	 e) Add 'MetadataUpdated' field to 'api.Container'
```

Comment:
nit: this should be space instead of tab.

Reply:
fixed.